### PR TITLE
DOCSP-44977 notes index build behavior during sync

### PR DESCRIPTION
**Pull request by jwilson-mdb**
## DESCRIPTION
Adds note to FAQ stating index builds on destination clusters are treated as writes

## STAGING
https://deploy-preview-463--docs-cluster-to-cluster-sync.netlify.app/faq/#can-i-perform-reads-or-writes-to-my-destination-cluster-while-mongosync-is-syncing-

## JIRA
https://jira.mongodb.org/browse/DOCSP-44977

## SELF-REVIEW CHECKLIST

- [ ] Does each file have 3-5 taxonomy facet tags?
 See the [taxonomy tagging instructions](https://wiki.corp.mongodb.com/display/DE/Taxonomy+tagging+instructions) and this [example PR](https://github.com/10gen/cloud-docs/pull/5042/files)
- [x]  Is this free of any warnings or errors in the RST?
- [x]  Is this free of spelling errors?
- [x]  Is this free of grammatical errors?
- [x]  Is this free of staging / rendering issues?
- [x]  Are all the links working?

## EXTERNAL REVIEW REQUIREMENTS
[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

**Review by jason-price-mongodb — APPROVED** [review]
@jwilson-mdb LGTM. Cheers.

File: source/faq.txt [hunk 1]
```diff
@@ -30,6 +30,11 @@ If you write to a synced namespace before issuing a :ref:`commit
 undefined. To ensure that you don't write to any synced namespaces, enable 
 :ref:`write blocking <c2c-write-blocking>`.
 
+.. note::
+   
+   Index builds on the destination cluster are treated as writes 
+   while ``mongosync`` is syncing.
```
Comment (mvankeulen94) [inline source/faq.txt:36]: CC @ajayvijayakumar123 this was added in response to the discussion around rolling index builds. Please LMK if there's anything else that needs to be added - thanks!

Comment (ajayvijayakumar123) [reply source/faq.txt:36]: this sounds good!

**Review by mvankeulen94 — APPROVED** [review]
lgtm, thanks!

**Pull request merged**